build/detect-secrets.js

```diff
@@ -17,10 +17,10 @@ const colors = require('colors');
 
 // Enum for exit codes
 const ExitCodes = {
-	SUCCESS: 0, 				        // success
+	SUCCESS: 0,                         // success
 	FOUND_SECRETS_OR_BASELINE_ISSUE: 1, // detect-secrets-hook found secrets or encountered a baseline file issue
 	DETECT_SECRETS_WRAPPER_ERROR: 2,    // an error occurred in this script
-	DETECT_SECRETS_ERROR: 3,            // an error occurred in detect-secrets
+	DETECT_SECRETS_ERROR: 999,          // an error occurred in detect-secrets (detect-secrets status code is appended)
```

Comment:
What's the reason for updating this to `999`? Is `999+n` a special status code for the hygiene checks?

Reply:
Previously, I was using `DETECT_SECRETS_ERROR: 3` as a prefix for the actual status code returned by `detect-secrets`, but I was using the format `<prefix>_<detect-secrets-code>` (e.g. `3_1`) which is not an integer and thus invalid to return.

So `999` is just to make it more visually clear in the format `<prefix><detect-secrets-code>`, e.g. `9991`

Reply:
`detect-secrets` also uses the status codes 0 and 1 and least, and I wanted to differentiate between this wrapper script returning status code 1 and `detect-secrets` returning status code 1 (which would be returned as 9991 by the wrapper)

Reply:
Can exit codes be that high? I get that it's a string here but eventually, it becomes an 8-bit integer, right?

Reply:
Haha yeah maybe they can't be that high :) It seems like the limit for a status code might be 255? 

I just set it back to `3` and removed the appending of the `detect-secrets` status code. The console error output and debug prints will have that extra information if needed.

Thanks for checking this!!